rootfs/var/lib/dietpi/services/fs_partition_resize.sh

```diff
@@ -42,25 +38,30 @@ w
 Y
 _EOF_
 
-#Everything else
-else
+	#Everything else
+	else
 
-	cat << _EOF_ | fdisk $TARGET_DEV
+		cat << _EOF_ | fdisk $TARGET_DRIVE
 p
 d
 $TARGET_PARTITION
 n
 p
 $TARGET_PARTITION
-$(parted $TARGET_DEV -ms unit s p | grep ':ext4::;' | sed 's/:/ /g' | sed 's/s//g' | awk '{ print $2 }')
+$(parted $TARGET_DRIVE -ms unit s p | grep ':ext4::;' | sed 's/:/ /g' | sed 's/s//g' | awk '{ print $2 }')
 
 p
 w
 
 _EOF_
 
+	fi
+
 fi
 
-partprobe $TARGET_DEV
+partprobe $TARGET_DRIVE
```

Comment:
This won't work for MMC?
```
root@DietPi:~# TARGET_DEV=$(findmnt / -o source -n)
root@DietPi:~# TARGET_PARTITION=${TARGET_DEV##*[a-z]} # Last [0-9]
root@DietPi:~# [[ $TARGET_DEV =~ mmcblk ]] || [[ $TARGET_DEV =~ nvme ]] && TARGET_DRIVE=${TARGET_DRIVE%[a-z]}
root@DietPi:~# echo $TARGET_DEV
/dev/mmcblk1p2
root@DietPi:~# echo $TARGET_PARTITION
2
root@DietPi:~# echo $TARGET_DRIVE

root@DietPi:~#
```

Reply:
@Fourdee 
With `TARGET_DRIVE=${TARGET_DEV%[0-9]}` in the first place it does:
```
2018-07-22 14:15:27 root@micha:/tmp# TARGET_DEV=$(findmnt / -o source -n)
2018-07-22 23:59:23 root@micha:/tmp# TARGET_PARTITION=${TARGET_DEV##*[a-z]} # Last [0-9]
2018-07-22 23:59:33 root@micha:/tmp# TARGET_DRIVE=${TARGET_DEV%[0-9]} # EG: /dev/mmcblk[0-9]p
2018-07-23 00:00:19 root@micha:/tmp# [[ $TARGET_DEV =~ mmcblk ]] || [[ $TARGET_DEV =~ nvme ]] && TARGET_DRIVE=${TARGET_DRIVE%[a-z]} # EG: /dev/mmcblk[0-9]
2018-07-23 00:00:26 root@micha:/tmp# echo $TARGET_DRIVE
/dev/mmcblk0
```
First we remove ending number, as this would be partition in every case. In case of mmc/nvme we then also remove the next last letter, to have correct `/dev/mmcblk0`.

Reply:
@MichaIng 

Apologies, old man moment 👴 Merge when ready 👍 
```
root@DietPi:~# echo $TARGET_DEV
/dev/mmcblk1p2
root@DietPi:~# echo $TARGET_DRIVE
/dev/mmcblk1
```